src/components/GrafanaContent.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from "react";
```

Comment:
importing React here isn't needed anymore (see https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html )